backends/discord/discord.cpp

```diff
@@ -0,0 +1,72 @@
+/* ScummVM - Graphic Adventure Engine
```

Comment:
Could we call it probably not "backends/discord" but "backends/presence" and the Discord. The current API looks more or less generic, so it could be potentially extended with other things like Facebook status 😉

Reply:
Will do!